src/ontology/cl-edit.owl

```diff
@@ -27848,6 +27880,30 @@ SubClassOf(obo:CL_4023050 obo:CL_4023040)
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0001384))
 SubClassOf(obo:CL_4023050 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_10090))
 
+
+# Class: obo:CL_4023051 (vascular leptomeningeal like cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) obo:IAO_0000115 obo:CL_4023051 "A type of mesothelial fibroblast that is derived from the neural crest and is a key component of the pia and arachnoid membranes surrounding the brain.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023051 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023051 "ILX:0770143")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "VLMC")
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_4023051 "vascular leptomeningeal cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:20023653"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:30096314"^^xsd:string) rdfs:comment obo:CL_4023051 "These enteric mesothelial fibroblasts (ENMFBs) are derived from the neural crest as evident from obtaining them by sorting Wnt1-Cre;R26Tomato cells (Tomato florescence labelled neural crest derived cells).")
```

Comment:
This is referring to a related cell type in a completely different location: "The fourth VLMC-like cell type (enteric mesothelial fibroblasts; ENMFBs) expressed all the VLMC marker genes but was found exclusively in the _enteric nervous system_." [PMID:20023653](https://www.cell.com/cell/fulltext/S0092-8674(18)30789-X?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS009286741830789X%3Fshowall%3Dtrue). (Note - not PMID:20023653 which is just about generation of markers. Refs should be sufficient for readers to find the source of assertions - as in any academic text.)

Looking over PMID:20023653 I think we probably need a general, species neutral term for mesothelial fibroblast of the leptomeninx (UBERON_0000391 = pia + arachnoid).  This would cover VLMC and other cell types mentioned in PMID:20023653 , e.g. ABC.  We should have species neutral terms for VLMC and ABC.  We can then look at adding species-specific terms with mouse markers and discuss whether we need a VLMC-like grouping class with explicit transcriptomics based def. 

Does that make sense?

Reply:
oh that makes more sense, i was wondering about that, thought it was some derived inference sort of situation. 
Currently they are species neutral, but I guess I should not add the markers in given that they were from mouse data. I've already added mesothelial fibroblast, but will add one specific to the leptomeninx under it. Happy to add a non species specific without the marker information, and add makers in another species specific entry. I'll also find better references. 

Things to do:

- [ ] Remove comment 
- [ ] Create mesothelial fibroblast of the leptomeninx & organise accordingly
- [ ] Remove markers from non species-specific term 
- [ ] Create species-specific entries and add above markers to them instead
- [ ] Fix references

Reply:
@dosumis think I fixed it - would like a review of it. Thanks